.github/workflows/test.yml

```
@@ -13,14 +13,24 @@ env:
 jobs:
   integration:
     name: Integration test
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+        - "3.13"
+        - "3.14"
+        os: [windows-latest, macos-latest, ubuntu-latest]
```

```suggestion
        python-version: ["3.13", "3.14"]
        os: [windows-latest, macos-latest, ubuntu-latest]
```
Shorter and consistent with the `os`?